File: leetcode/217. Contains Duplicate/ContainsDuplicate.cpp
class Solution
{
public:
    bool containsDuplicate(vector<int> &nums)
    {
        map<int, int> m;
        for (const int num : nums)
        {
            if (m.find(num) != m.end())
            {
                return true;
            }
            else
            {
                m[num]++;
            }
        }
        return false;
    }
};

File: leetcode/217. Contains Duplicate/ContainsDuplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function (nums) {
    const m = new Map();
    for (let i of nums) {
        if (m.has(i)) {
            return true
        } else {
            m.set(i);
        }
    }
    return false;
};